app/controllers/return-logs-setup.controller.js

```diff
@@ -43,6 +52,25 @@ async function start(request, h) {
   return h.view('return-logs/setup/start.njk', pageData)
 }
 
+async function submitMeterProvided(request, h) {
+  const {
+    params: { sessionId },
+    payload
+  } = request
```

Comment:
This is more of my preference, I'm not really a fan of the way this is written as it's a bit harder to follow: ` const {
    params: { sessionId },
    payload
  } = request`
  
I prefer `const { sessionId } = request.params` and then passing the `request.payload` to the service. I just find that easier to read. I see that it has been done that way in some other controllers so I guess it's fine to do it like that.